sys/vm/vm_cheri_revoke.c

```diff
@@ -867,7 +866,7 @@ vm_cheri_revoke_pass(const struct vm_cheri_revoke_cookie *crc, int flags)
 	 * we'll either trap on them belatedly and fix them up
 	 * (PMAP_CAPLOADGEN_ALREADY), or they'll get shot down when we
 	 * increment the CLG for the *next* pass (while we've got the world
-	 * stopped.)
+	 * stopped.) XXX-MJ is this true in the load-side revoker?
```

Comment:
I believe so, yes.  The comment might want to reference `pmap_caploadgen_next` as
https://github.com/CTSRD-CHERI/cheribsd/pull/2011/commits/da607d5877f2593b32044d754b5a1349db2d6aa0#diff-f8d48857b9862425852298f53461c36ca8a6e968a1399b12f4d1d3e594470de1R583 does.